### PR TITLE
feat(TMRX-1555): refactor content bucket 1

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
### Description

We are seeking to refactor our slices to improve the loading state and simplify the associated code to make it easier to read and update.

Technical implementation:

Refactor Content Bucket 1 component to reduce the dependency on shared slice components, where applicable. This will allow us to reduce complexity from additional logic and props being used to accommodate the various slices.

Use media queries and/or Visible / Hidden Newskit components instead of client side useBreakpointKey hook

Acceptance Criteria

The loading state of the component should begin immediately on page load and should match the structure of the slice once it is fully loaded on all breakpoints

The loaded in slice should match the current implementation.

[TMRX-1555](https://nidigitalsolutions.jira.com/browse/TMRX-1555)

### Checklist

- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

[TMRX-1555]: https://nidigitalsolutions.jira.com/browse/TMRX-1555?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/slices/content-bucket-1/index.tsx [hunk 1]
```diff
@@ -36,96 +29,89 @@ export const ContentBucket1 = ({
...
+          />
+          <LeadArticle
+            article={{
+              ...leadArticle,
+              hasTopBorder: true,
+              imageTop: false,
+              contentWidth: {
+                md: '283px',
+                xl: '312px'
+              }
```
[the remark this answers is not in the document]
Comment (douglasmik) [reply packages/ts-newskit/src/slices/content-bucket-1/index.tsx:70]: good spot - must've gotten undone on rebase. Updating now

File: packages/ts-newskit/src/components/slices/article/index.tsx [hunk 2]
```diff
@@ -130,29 +142,19 @@ export const Article = ({
...
-              }}
-            />
+            articleDivider
           )}
         </GridLayoutItem>
       )}
       {showImage ? (
         isLeadImage ? (
+          /* @ts-ignore */
           <FullWidthCardMediaMob {...cardImage} />
```
[the remark this answers is not in the document]
Comment (douglasmik) [reply packages/ts-newskit/src/components/slices/article/index.tsx:154]: This should only be on article images, where lead articles are handled in a different component (packages/ts-newskit/src/components/slices/lead-article/index.tsx). Probably more a case of our naming here I think. Checked and only article images are receiving `loading` attribute.

[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/components/slices/article/index.tsx:154]: Why does it have the option of isLeadImage though?

File: packages/ts-newskit/src/components/slices/article/index.tsx [hunk 3]
```diff
@@ -130,29 +142,19 @@ export const Article = ({
...
-                stylePreset: 'dashedDivider'
-              }}
-            />
+            articleDivider
           )}
         </GridLayoutItem>
       )}
       {showImage ? (
         isLeadImage ? (
+          /* @ts-ignore */
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/slices/article/index.tsx:153]: What's the TS error here?

File: packages/ts-newskit/src/components/slices/article/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (douglasmik) [reply packages/ts-newskit/src/components/slices/article/index.tsx:154]: think it refers to the 'lead' images in StackedModule and similar slices - it's not the `LeadArticles`

[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/components/slices/article/index.tsx:154]: Oh I see - in that case it could apply to all of those then I get you

**Review by bibicollins1 — APPROVED** [review]

**Pull request merged**